types/eip4844/block.yaml

```diff
@@ -0,0 +1,95 @@
+Eip4844:
+  BeaconBlockBodyCommon:
+    # An abstract object to collect the common fields between the BeaconBlockBody and the BlindedBeaconBlockBody objects
+    type: object
+    description: "The [`BeaconBlockBody`](https://github.com/ethereum/consensus-specs/blob/master/specs/eip4844/beacon-chain.md#beaconblockbody) object from the CL EIP-4844 spec."
+    properties:
+      randao_reveal:
+        allOf:
+          - $ref: '../primitive.yaml#/Signature'
+          - description: "The RanDAO reveal value provided by the validator."
+      eth1_data:
+        $ref: '../eth1.yaml#/Eth1Data'
+      graffiti:
+        $ref: '../primitive.yaml#/Graffiti'
+      proposer_slashings:
+        type: array
+        items:
+          $ref: '../proposer_slashing.yaml#/ProposerSlashing'
+      attester_slashings:
+        type: array
+        items:
+          $ref: '../attester_slashing.yaml#/AttesterSlashing'
+      attestations:
+        type: array
+        items:
+          $ref: '../attestation.yaml#/Attestation'
+      deposits:
+        type: array
+        items:
+          $ref: '../deposit.yaml#/Deposit'
+      voluntary_exits:
+        type: array
+        items:
+          $ref: '../voluntary_exit.yaml#/SignedVoluntaryExit'
+      sync_aggregate:
+        $ref: '../altair/sync_aggregate.yaml#/Altair/SyncAggregate'
+      blob_kzg_commitments:
+        $ref: '../primitive.yaml#/KZGCommitment'
```

Comment:
This is `List[KZGCommitment, MAX_BLOBS_PER_BLOCK]`. What do you think of following a similar convention of creating a new file on the types folder for the `blob_kzg_commitments`, like [bls_to_execution_change.yaml](https://github.com/ethereum/beacon-APIs/blob/master/types/bls_to_execution_change.yaml)?

Reply:
Ah good catch thanks, updated `type` to array. I actually started with creating a new file for  `blob_kzg_commitments ` but realised I only needed a primitive type, and looking at previous changes I guessed the convention is to add primitive types to `primitive.yaml`, but I could be completely wrong 😅